dataprofiler/profilers/profiler_options.py

```diff
@@ -376,6 +376,44 @@ def _validate_helper(self, variable_path='IntOptions'):
         return super()._validate_helper(variable_path) 
 
 
+class PrecisionOptions(BooleanOption):
```

Comment:
Not saying this is a problem, but just something to note, in `profiler_options`, some of them are plural (e.g. NumericalOptions) and some are singular (e.g. BooleanOption). It was my understanding that the singular ones were for smaller scale options that exist in other options (boolean, histogram, etc) and plural was for data type columns (float, int, text, etc). Whether or not that is correct, this may be something we should address as given how complicated the options are already, this naming inconsistency doesn't help either. Does anyone have any insight or thoughts on this to explain, or is it an oversight that I am applying unintended meaning to?